base/src/main/scala/data/DisjunctionFunctions.scala

```diff
@@ -1,10 +1,17 @@
 package scalaz
 package data
 
+import Disjunction.{\/, \/-, -\/}
+
 trait DisjunctionFunctions {
-  @inline def -\/[L](value: L): Disjunction[L, Nothing] = Disjunction.-\/(value)
-  @inline def \/-[R](value: R): Disjunction[Nothing, R] = Disjunction.\/-(value)
+  @inline def -\/[L](value: L): Disjunction[L, Nothing] = -\/(value)
```

Comment:
Meh, I've come around to finding it convenient for `\/`, because an expression typically fixes only one side, and the variance lets the unification be delayed. As long as you don't want to add any methods with higher-kinded args—i.e kill it as soon as it gets in the way of anything.